auto-cpufreq-installer

```diff
@@ -125,12 +126,21 @@ function tool_install {
 
   header "Installing necessary Python packages"
 
+<<<<<<< HEAD
+  venv_dir=/opt/auto-cpufreq/venv
+  mkdir -p "${venv_dir}"
+  python3.10 -m venv "${venv_dir}"
+
+  source "${venv_dir}/bin/activate"
+  python3.10 -m pip install --upgrade pip wheel
```

Comment:
Why would you want to limit it to 3.10 in particular, especially since latest version is 3.12 when otherwise it'll just use latest version?